api/controllers/v1/jobs.ts

```diff
@@ -165,13 +166,40 @@ async function NotifyBuildSummary(jobId: string): Promise<any> {
   await client.connect();
   const db = client.db(c.get('dbName'));
   const env = c.get<string>('env');
+  const githubToken = c.get<string>('githubBotPW');
 
   const jobRepository = new JobRepository(db, c, consoleLogger);
   // TODO: Make fullDocument be of type Job, validate existence
   const fullDocument = await jobRepository.getJobById(jobId);
+  if (!fullDocument) {
+    consoleLogger.error(jobId, 'Cannot find job entry in db');
+    return;
+  }
   const repoName = fullDocument.payload.repoName;
   const username = fullDocument.user;
+  const githubCommenter = new GithubCommenter(consoleLogger, c, githubToken);
```

Comment:
Just a heads up we'll need to remove `c` to ensure `GithubCommenter` is using the token correctly. This should probably be applied to the instances declared in the tests too. 

(I'm surprised the linter didn't pick this up!)

Reply:
ah jeez, that was a bad mistake. Thank you

Reply:
Alllll good 😎